code/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" href="https://i.ibb.co/ctPyhfL/burger.png">
+    <!-- dont forget the meta tag for responsiveness -->
+  </head>
+
+  <body>
+   <header>
+      <!-- Some kind of logo and navbar -->
+    <a href="./index.html">
+      <img class="logo" src="./pixelburger.png" alt="burger">
+    </a>
+      <nav class="navigation">
+        <li class="nav-item">
+        <a href="./about.html">ABOUT</a>
+      </li>
+      <li class="nav-item">
+        <a href="./contact.html">CONTACT</a>
+      </li>
+      </nav>
+  </header>
+
+    <section class="big-news">
+            <!-- The big news content -->
+        <img class="contact" src="https://i.ibb.co/pP4jXsH/Screenshot-20210825-231722-Instagram.jpg">
+      <div class="big-text">
+        <h3>CONTACT</h3>
+        <h4>pmattsson89@gmail.com
+        <br>@mattsson717
```

Comment:
<br> for new line of text is supposedly "old school" html and should be avoided. A new line of <p> should to the trick as well maybe?